README.md

```diff
@@ -1,11 +1,41 @@
 Non-Fungible Tokens (NFTs)
 ==========================
 
+[![Open in Gitpod!](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/near-examples/NFT)
+
+<!-- MAGIC COMMENT: DO NOT DELETE! Everything above this line is hidden on NEAR Examples page -->
+
 This repository includes NFT implementations in Rust and AssemblyScript for [NEP#4](https://github.com/nearprotocol/NEPs/pull/4)
 
+
+
 Rust
 ====
 
+1. Prerequisites:
+
+If you are using Gitpod, you can skip this section! Your environment is already set up 🎉
+
+  * Make sure Rust is installed per the prerequisites in [`near-sdk-rs`](https://github.com/nearprotocol/near-sdk-rs)
+  * Make sure you have node.js installed (we like [asdf](https://github.com/asdf-vm/asdf) for this),  then use it to install [yarn]: `npm install --global yarn` (or just `npm i -g yarn`)
```

Comment:
I think Chad suggested using asdf at one point and it turns out it's not gonna work with Windows. Can we just point them to the regular NodeJS installation page?